inetum-realjobs-frontend/src/app/app.routing.ts

```diff
@@ -5,13 +5,15 @@ import {FilterFormComponent} from "./components/filter-form/filter-form.componen
 import {AuthGuardService} from "./guard/authGuard.service";
 import {NotFoundComponent} from "./components/not-found/not-found.component";
 import {RegisterComponent} from "./components/register/register.component";
+import {VacancyDetailsComponent} from "./components/vacancy-details/vacancy-details.component";
 
 const appRoutes: Routes = [
   {path: "", redirectTo: "/vacancy/search", pathMatch: "full"},
   {path: "users/:username", component: UserDetailsComponent, canActivate: [AuthGuardService]},
   {path: "login", component: LoginComponent},
   {path: "register", component: RegisterComponent},
   {path: "vacancy/search", component: FilterFormComponent},
+  {path: "vacancy/:id", component: VacancyDetailsComponent},
```

Comment:
I don't think it is a good idea to display the id in the URL

Reply:
Judged to be the best way.